### PR TITLE
Resolve bug showing active style and add subgroup to email

### PR DESCRIPTION
**Pull request by norbye**
# Description

Added subgroups to email-user list view - as some of them have @abakus.no emails as well.

Also fixed a 'lil bug I introduced to the `Table` component way back when I refactored it that made the filter stuff look active even when it wasn't. Now it works the way it's supposed to again, properly tested(:

# Result

If you've made visual changes, please check the boxes below and include images showing the changes. Descriptions are appreciated.

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

# Testing

- [x] I have thoroughly tested my changes.

Please describe what and how the changes have been tested, and provide instructions to reproduce if necessary.

---

Resolves ... (either GitHub issue or Linear task)

### CODE REVIEW

File: app/routes/admin/email/components/EmailUsers.tsx [hunk 1]
```diff
@@ -35,15 +42,16 @@ const EmailUsers = () => {
...
-  );
+
+  const relevantGroups = [
+    ...useAppSelector((state) =>
+      selectGroupsByType(state, GroupType.Committee),
+    ),
+    ...useAppSelector((state) => selectGroupsByType(state, GroupType.Board)),
+    ...useAppSelector((state) => selectGroupsByType(state, GroupType.Ordained)),
+    ...useAppSelector((state) => selectGroupsByType(state, GroupType.SubGroup)),
+  ];
```
Comment (ivarnakken) [inline app/routes/admin/email/components/EmailUsers.tsx:53]: Wouldn't it be better to create a new selector for this?

File: app/components/Table/HeadCell.tsx [hunk 2]
```diff
@@ -213,11 +207,11 @@ const HeadCell: React.FC<HeadCellProps> = ({
               <Icon
                 name="options"
                 size={16}
-                className={cx(
+                className={
                   filters[filterIndex] !== undefined || isShown[filterIndex]
```
Comment (ivarnakken) [inline app/components/Table/HeadCell.tsx:211]: Isn't this basically the same as checking for `iconIsActive`?

**Review by ivarnakken — APPROVED** [review]

File: app/routes/admin/email/components/EmailUsers.tsx (hunk 1 above)
Comment (norbye) [reply app/routes/admin/email/components/EmailUsers.tsx:53]: You do have a very good point

Comment (norbye) [reply app/routes/admin/email/components/EmailUsers.tsx]: Do you have any comments on the changes to the selector @ivarnakken?

**Pull request merged**